src/crypto/byron/util.test.js

```diff
@@ -273,8 +273,8 @@ describe('signTransaction', () => {
     // If this fails, it means we let this bug open for too long time.
     // Try updating rust library and seeing if it disappears
     // Note(v-almonacid): I increased this timer. We'll hopefully update the
-    // bindings within the the next months and this issue should be fixed by then©
-    expect(moment().isBefore('2019-12-01')).toBeTruthy()
+    // bindings within the the next months and this issue should be fixed by then
+    expect(moment().isBefore('2019-12-12')).toBeTruthy()
```

Comment:
@v-almonacid I would aim to 15 or something like that

Reply:
Actually it's going to be more because at the time I wrote that comment we were thinking in going from v1 to v2 but now there is no much sense on doing that, and I assume we'll keep using v1 and v3 and eventually just v3.